content/en/docs/how-to/registry-authentication.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Authenticate with OCI-compliant registries"
+description: "How to use notation to authenticate with OCI-compliant registries"
+type: docs
+weight: 1
+---
+
+You can use `notation` to authenticate to an OCI-compliant registry. This is useful for pushing and pulling signed artifacts, as well as inspecting artifacts in the registry.
```

Comment:
This text:
> This is useful for pushing and pulling signed artifacts, as well as inspecting artifacts in the registry.

Is a bit confusing. Are we saying that users cannot do that if they are not authenticated? We should add clarification like "... in registries that do not offer public access" or something like that.

Reply:
good call @toddysm 

Incorporated you feedback